docs/reference/ml/df-analytics/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -213,6 +213,10 @@ belongs.
   `multiclass_confusion_matrix`:::
     (Optional, object) Multiclass confusion matrix.
 
+    `size`::::
+      (Optional, double) Specifies the size of the multi class confusion matrix. 
```

Comment:
Should we spell it "multi class" or "multiclass" like in line 214?

Reply:
Typo. Fixed in [977659f](https://github.com/elastic/elasticsearch/pull/85735/commits/977659f0d9e5baee09d4595696d72c240ee3e654). Thanks!